grafana-tidb-dashboard.md

```diff
@@ -38,17 +38,16 @@ aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-mon
     - CPU Usage：每个 TiDB 实例的 CPU 使用统计
     - Connection Count：每个 TiDB 的连接数
     - Open FD Count：每个 TiDB 实例的打开的文件描述符统计
+    - Disconnection Count： 每个 TiDB 实例断开连接的数量
+    - Event OPM：每个 TiDB 实例关键事件统计，例如 start，close，graceful-shutdown，kill，hang 等
     - Goroutine Count：每个 TiDB 实例的 Goroutine 数量
-    - Go GC Duration：每个 TiDB 实例的 Golang GC 耗时
-    - Go Threads：每个 TiDB 实例的线程数量
-    - Go GC Count：每个 TiDB 实例的 Golang GC 执行次数
-    - Go GC CPU Usage：每个 TiDB 实例的 Golang GC 使用的 CPU
```

Comment:
Some of these metrics are removed. Do we still need these metrics? @jackysp

Reply:
I think these are integrated into other indicators, such as PR https://github.com/pingcap/tidb/pull/20683